Fix:

libscream.c:3231:16: error: implicit declaration of function 'safe_print_string' [-Werror,-Wimplicit-function-declaration]

https://github.com/mej/Eterm/pull/2
--- src/libscream.c.orig	2020-03-03 11:33:47.000000000 -0600
+++ src/libscream.c	2021-04-27 13:32:59.000000000 -0500
@@ -44,6 +44,7 @@
 
 #include "config.h"
 #include "feature.h"
+#include "misc.h"
 
 /* use libast if we have it */
 #ifdef DEBUG_ESCREEN
